app/views/catkeys/_edit.html.erb

```diff
@@ -1,10 +1,43 @@
-<%= form_with model: @change_set, url: item_catkey_path(@change_set), method: :patch do |f| %>
-  <div class="mb-3">
-    <%= f.label :catkey %>
-    <%= f.text_field :catkey, class: 'form-control' %>
-    <p class='form-text'>
-      Separate multiple catkeys with commas.
-    </p>
-  </div>
-  <button class="btn btn-primary">Update</button>
+<%= render EditModalComponent.new do |component| %>
+  <% component.header { 'Manage catkey' } %>
+  <% component.body do %>
+    <div class="mb-3 row plain-container">
+      <div class="col-sm-2"></div>
+      <div class="col-sm-10 col-xl-8">
+        <strong>Catkey</strong>
+      </div>
+      <div class="col-sm-1"></div>
+    </div>
+    <%= form_with model: @form,
+                      url: item_catkey_path(@form),
+                      method: :patch,
+                      data: { controller: 'nested-form', nested_form_selector_value: '.plain-container' } do |form| %>
```

Comment:
I don't think this is an appropriate controller, because nested-form depends on the child resources actually being resources (like with an identifier) rather than parts of a bigger record.